src/neo-vm/Types/Null.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Neo.VM.Types
+{
+    internal class Null : StackItem
+    {
+        public override bool Equals(StackItem other)
+        {
+            if (ReferenceEquals(this, other)) return true;
+            if (other is null) return true;
+            if (other is Null) return true;
+            return false;
+        }
+
```

Comment:
If PUSHNULL is similar to PUSH0 (position 0x50), shouldn't this be conversible to byte[0] and BigInteger(0)?